### ISSUE TITLE
Badge / Pathway RFC process and template

### ISSUE THREAD

**Issue by amiller-gh** [nodejs/badges#4]
We need a process to evaluate and accept new badge and pathway proposals. I recommend we use an RFC-like process for new additions. Every new badge or pathway proposal will be submitted as a PR to add a new file to the @nodejs/open-badges repo by filling out a template markdown file with the required information. Discussion will happen on the PR and when consensus is reached by the Open Badges Team, the PR will be merged

Outstanding questions:

What information do we need in the RFC template?
Do we require a final design before merging the PR? If no, what is the design process?
Does the larger Community Committee want final say on badge approval?
What is the process for modifying badge designs / criteria?
Is there a better format / process for proposing new Badges?

I welcome anyone to take a stab at a first draft and opening a PR for further discussion.

**Comment by amiller-gh** [comment]
Good example of a modern RFC repo: https://github.com/emberjs/rfcs

**Comment by maddhruv** [comment]
I feel we should collect pieces of information regarding -
- The purpose behind the badge
- Impact of issuing the badge
- Governance of the badge issuer
- Keeping track of the badge and its usage

**Comment by maddhruv** [comment]
Yeah, we can set up some design principles with a designing badge so that we can follow up a standard throughout the scenario.

Taking up a final decision before merging or approving a PR should be appreciated.
